EDQueue.podspec

```diff
@@ -1,14 +1,14 @@
 Pod::Spec.new do |s|
   s.name         = 'EDQueue'
-  s.version      = '0.7.1'
+  s.version      = '0.7.3'
```

Comment:
If we are going to break backwards compatibility (min platform to iOS 7) we should bump the major version number as per semver. That would make this `1.0.0`.